recipes/jupyter_contrib_nbextensions/post-link.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+:: We redirect stderr & stdout to conda's .messages.txt; for details, see
+::     http://conda.pydata.org/docs/building/build-scripts.html
+"%PREFIX%\Scripts\jupyter.exe" contrib nbextension install --sys-prefix >> "%PREFIX%/.messages.txt" 2>&1
```

Comment:
Maybe `contrib nbextension` -> `contrib_nbextension` here and for the rest of them?